pxtpy/converter.ts

```diff
@@ -204,21 +204,24 @@ namespace pxt.py {
         return fillTypes(lookupApi(name))
     }
 
-    function initApis(apisInfo: pxtc.ApisInfo) {
+    function initApis(apisInfo: pxtc.ApisInfo, tsShadowFiles: string[]) {
         internalApis = {}
         externalApis = {}
 
+        let tsShadowFilesSet = U.toDictionary(tsShadowFiles, t => t)
         for (let sym of U.values(apisInfo.byQName)) {
-            // sym.pkg == null - from main package - skip these
-            if (sym.pkg != null) {
-                let sym2 = sym as SymbolInfo
+            // if the symbol comes from a .ts file that matches one of the .py files we're compiling - skip these
+            if (sym.fileName in tsShadowFilesSet) {
```

Comment:
@mmoskal , instead of filtering by package, I filter per file. E.g. filter out symbols from a.ts if a.py exists.

Reply:
we usually don't use `in`, since `"toString" in {} == true`; use `hasOwnProperty`.

Reply:
don't we typically do ``!!tsShadowFileSet[sym.fileName]`` ? (assuming non-empty file names)

Reply:
I guess it's fine for file names, but `!!({})["toString"] == true`